File: C/Iniciante/1009.c
#include <stdio.h>
 
int main() {
 
    char nome[50];
    double salario, vendas, bonus, salarioTotal;
    
    printf("");
    scanf("%s", &nome);
    printf("");
    scanf("%lf", &salario);
    printf("");
    scanf("%lf", &vendas);
    
    bonus = vendas * 0.15;
        
    salarioTotal = salario + bonus;
 
    printf("TOTAL = R$ %.2lf\n", salarioTotal);
}

File: C#/Iniciante/1009.cs
using System; 

class URI {

    static void Main(string[] args) { 

        string nome = Console.ReadLine();
        double salario = double.Parse(Console.ReadLine());
        double valorVendido = double.Parse(Console.ReadLine());
        
        double bonus = valorVendido * 0.15;
        
        double salarioFinal = salario + bonus;
        
        Console.WriteLine($"TOTAL = R$ {salarioFinal:F2}");

    }

}